docs/src/api.rst

```diff
@@ -13,7 +13,7 @@ Functions
 =========
 
 .. autosummary::
-   :toctree: generated/
+   :toctree: ./generated/api/functiion/
```

Comment:
```suggestion
   :toctree: ./generated/api/function/
```

Reply:
I'm a bit naive here, but what is the point of separating functions from classes ?
I'd like to have a flat organization, thus not having levels that are not primarily relevant.

Reply:
This organisation will change in the future. 
There is no point for this library to have a separation between functions and classes.
I am just waiting that the new API and the focus of the library are fixed to propose a better organisation of the API.